api/internal/filter/authentication.go

```diff
@@ -45,7 +45,7 @@ func (mw *AuthenticationMiddleware) Handle(ctx droplet.Context) error {
 
 	req := httpReq.(*http.Request)
 
-	if req.URL.Path == "/apisix/admin/tool/version" || req.URL.Path == "/apisix/admin/user/login" {
+	if req.URL.Path == "/apisix/admin/tool/version" || req.URL.Path == "/apisix/admin/user/login" || req.URL.Path == "/apisix/admin/cache_verify" {
```

Comment:
need auth for the new API

Reply:
what's the proper way to use this API in cache_verify command?
is the current implementation desirable?
so we just need to first get token,then use this API this way?